ccm_web/client/src/pages/AddUMUsers.tsx

```diff
@@ -92,7 +92,7 @@ function AddUMUsers (props: AddUMUsersProps): JSX.Element {
   const classes = useStyles()
```

Comment:
1. Pick a section( say `section 1`)
2. Upload a CSV file with one valid uniqname (say `studenta`) and others with invalid uniqname (say `testa, testb`) roles as students. 
2. After the submit, it will error saying `[testa, testb]` not valid users and `studenta` is success at this point
3. When you hit `Back` button, then section calls should be fetch again. Since it won't show the newly added student ( with count indicator) nor if a new section is manually created then it won't show it in the list.

Reply:
Probably for the features where ever we allow users to pick a section, then using the Back button should fetch latest section.

Reply:
Thanks for flagging this! I have some ideas on how to handle it, I'll work on that today.

Reply:
As discussed, I have rewired `AddUMUsers.tsx`, `AddNonUMUsers.tsx` (and leaf components), and `FormatThirdPartyGradebook.tsx` so that they fetch the latest section data whenever the stage that uses that data appears, including on "Back" button clicks.